lib/nelder_mead.mli

```diff
@@ -0,0 +1,13 @@
+(** Minimizes cost function [f].
```

Comment:
There should be a toplevel comment for the module, I'll add it in a separate commit